lib/marked.js

```diff
@@ -872,7 +872,10 @@ Renderer.prototype.link = function(href, title, text) {
       return '';
     }
   }
-  var out = '<a href="' + href + '"';
+  // Navigate to any external links using the parent frame
+  var isExternalLink = href.startsWith("http");
```

Comment:
`startsWith` appears to be unsupported in IE:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith

Reply:
I can always count on IE to make my life more difficult. I changed to indexOf to make sure we get full browser support.